requirements.txt

```diff
@@ -1,3 +1,2 @@
-# 3.12.2 version is used as it is
-# the version available in alpine 3.12 (for protobuf cpp version)
-protobuf>=3.12.2,<4.0
+# Minimum of 3.19 as protoc useed needed it
+protobuf>=3.19.2
```

Comment:
are you sure it shouldn't be > 4.0? or does it matter? which version did you test?

Reply:
In this PR I updated the github action to use 3.20.3.
I cannot use a 4.x version as setup-protoc@v1 only support 3.x.x scheme

Reply:
I tried to used 3.21.9 for protoc, but it is not an official release so action doesn't work.

But it seems to work well even if we use protoc 3.20.3 and 3.21.9 libprotobuf in alpine.